docs/conf.py

```diff
@@ -12,8 +12,8 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+import configparser
```

Comment:
I think this will need (a) dropping Python 2.7 or (b) use of six or (c) two different imports for Python 2.7 and 3.x to work, see the note on top of https://docs.python.org/2.7/library/configparser.html .

Reply:
This is for building docs, not running the library. We can make the docs depend on Python 3. There isn't really a need to continue building docs on Python 2.

Reply:
Interesting point. I'll trust your evaluation.